### ISSUE TITLE
A11y issue : semantics on Lists being removed with Safari as list-style: none is a default.

### ISSUE THREAD

**Issue by sparky-raccoon** [tailwindlabs/tailwindcss#8975]
**What version of Tailwind CSS are you using?**
v3.1.6

**What build tool (or framework if it abstracts the build tool) are you using?**
React v18.2.0

**What version of Node.js are you using?**
v16.14.2

**What browser are you using?**
Safari v15.5

**What operating system are you using?**
macOS Monterey v12.4

**Reproduction URL**
Create a basic React app with CRA for instance, add a list `<ul>` with a few elements. Launch VoiceOver and go through the page content to find the list. VoiceOver does not announce any list and just reads each `<li>` element text. 

**Describe your issue**
Because `list-style: none` is applied as default by Tailwind, semantics are removed as aknowledged by different people working on accessibility (check this post : https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html).
I understand why native styles are removed as default (they are usually reset) and this is more a Safari issue. Yet, as tailwind reaches almost 3M downloads per week (https://www.npmjs.com/package/tailwindcss), this behaviour may have a significant impact. Of course, it's up to Safari developers to fix this on their side if they're willing to. Meanwhile, an alternative to default styles for lists should be provided, or at least something in the docs to notify tailwind users about this issue. 

One hack to restore semantics when `list-style: none` is applied : https://gerardkcohen.me/writing/2017/voiceover-list-style-type.html

**Comment by adamwathan** [comment]
Hey! Yep we know about this which is why is mentioned here in the docs:

https://tailwindcss.com/docs/preflight#accessibility-considerations

No plans to change anything here, at the end of the day it’s up to users to test their websites thoroughly and write the correct HTML.

**Comment by jiri-thiemel-te** [comment]
I don't agree with this argument. List's should be used only when they make sense for semantics. Otherwise you can use just divs.

In many scenarios developers use them for navigations, therefore screen reader users often navigate by lists.

It's very easy to reset a list and keep it's semantics even for Safari. Instead of `list-style: none` just do this:

```css
list-style: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg'/%3E");
```

The issues with current approach are:

1. Developers not knowing about this issue with Safari.
2. Having to redeclare the role is a huge concern - anyone can easily forget.
4. It's unnecessary, the solution above is tested and used by those that know about this issue - which is unfortunately a very small percentage of developers
